components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/select-attribute-values.jsp

```diff
@@ -247,14 +248,14 @@
                 createInputs(paths[i].name);
             }
         }
-        document.attributeValueForm.action = "<%=returnPage%>?category="
-                + '<%=category%>' +"&ruleId=" + '<%=ruleId%>' ;
+        document.attributeValueForm.action = "<%=Encode.forUriComponent(returnPage)%>?category="
```

Comment:
Here variable returnPage is not a component in query string. Encode that for javascript block